src/App.jsx

```diff
@@ -20,9 +22,9 @@ function App() {
       <Route path="/articles" element={ <Articles/>} />
       <Route path="/articles/:article_id" element={ <SingleArticleCard/>} />
        <Route path="/topics" element={<Topics />} />
-       {/* <Route path="/topics/topic=football" element={<FootballTopic />} />
-       <Route path="/topics/topic=cooking" element={<CookingTopic />} />
-       <Route path="/topics/topic=coding" element={<CodingTopic />} /> */}
+       {/* <Route path="/topics?topic=football" element={<TopicFootball />} /> */}
+       {/* <Route path="/topics?topic=cooking" element={<TopicCooking />} />
```

Comment:
I don't know what's going on here with the commented out components, but it's not necessary to have a separate component for each topic- - instead, your articles component should be able to take a topic prop.